### PR TITLE
Feature/#33 add map layout component

### LINKED ISSUE #33: Add Map Layout Component
- [ ] When in this view we see a Header with some Text and a Button to go to the `chat` view. 
- [ ] Then in the center, we see a Map and Address divs.
- [ ] At the bottom we see a Footer ver with Text and a Button to redirect to WhatsApp:
- [ ] Data can be passed as props

image

### PR DESCRIPTION
**Pull request by elpollo12**
### Description

Added map layout with styled components.

fixes # (issue)

#33 

#### Screenshots
scrnli_20_10_2021 10-11-06

### CODE REVIEW

File: src/pages/index.js [hunk 1]
```diff
@@ -25,6 +26,9 @@ const Pages = () => (
       <Route path="/confirm/:id?">
         <Confirm />
       </Route>
+      <Route path="/map/:id?">
+        <MapLayout />
+      </Route>
```
Comment (Israel-Laguan) [inline src/pages/index.js]: This route  changed to `delivery`. Please move the implementation to Delivery and erase this new route

**Review by Israel-Laguan — CHANGES_REQUESTED** [review]

File: src/pages/index.js (hunk 1 above)
Comment (Israel-Laguan) [reply src/pages/index.js]: Also a layout should be used inside the Page

File: src/components/Map/Map.styled.jsx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+  background-color: #486ef5;
+  border-top-left-radius: 25px;
+  border-top-right-radius: 25px;
+  width: 50%;
+  margin: 0 25%;
+  height: 20%;
+  display: flex;
+  flex-wrap: wrap;
+  @media (max-width: 960px) {
+    width: 80%;
```
Comment (Israel-Laguan) [inline src/components/Map/Map.styled.jsx]: If you use relative or dynamic with, please include min-width and max-width

File: src/components/Map/Map.styled.jsx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  }
+`
+
+const FooterMap = styled.div`
+  position: absolute;
+  bottom: 0;
+  background-color: #486ef5;
+  border-top-left-radius: 25px;
+  border-top-right-radius: 25px;
+  width: 50%;
```
Comment (Israel-Laguan) [inline src/components/Map/Map.styled.jsx]: If you use relative or dynamic with, please include min-width and max-width

File: src/components/Map/Map.styled.jsx [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  margin: 0 25%;
+  height: 20%;
+  display: flex;
+  flex-wrap: wrap;
+  @media (max-width: 960px) {
+    width: 80%;
+    margin: 0 10%;
```
Comment (Israel-Laguan) [inline src/components/Map/Map.styled.jsx]: The relative margin on widescreen looks weird, please verify on widescreens

File: src/components/Map/Map.styled.jsx [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+  }
+`
+
+const FooterMapAddress = styled.div`
+  flex: ${props => props.cols};
+  text-align: center;
+  padding: 5% 0;
```
Comment (Israel-Laguan) [inline src/components/Map/Map.styled.jsx]: The relative margin on widescreen looks weird, please verify on widescreens and in mobile

File: src/components/Map/Map.styled.jsx [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+
+const FooterMap = styled.div`
+  position: absolute;
+  bottom: 0;
+  background-color: #486ef5;
+  border-top-left-radius: 25px;
+  border-top-right-radius: 25px;
+  width: 50%;
+  margin: 0 25%;
+  height: 20%;
```
Comment (Israel-Laguan) [inline src/components/Map/Map.styled.jsx]: If you use relative or dynamic height, please include min-height and max-height

File: src/components/Map/Map.styled.jsx [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+
+const HeaderSubTitle = styled.div`
+  display: flex;
+  text-align: left;
+  font-size: 0.55em;
+  color: white;
+  margin: 0 auto;
+  :before {
+    content: '${props => props.subtitle}';
+  }
```
Comment (Israel-Laguan) [inline src/components/Map/Map.styled.jsx]: why not inside the div?

File: src/components/Map/Map.styled.jsx [hunk 8]
```diff
@@ -0,0 +1,119 @@
...
+`
+
+const HeaderTitle = styled.div`
+  display: flex;
+  text-align: left;
+  margin: 0 auto;
+  :before {
+    content: '${props => props.title}';
+  }
```
Comment (Israel-Laguan) [inline src/components/Map/Map.styled.jsx]: Why not inside the div

**Comment by elpollo12** [comment]
Updated code.

File: src/pages/index.js (hunk 1 above)
Comment (elpollo12) [reply src/pages/index.js]: Can I replace the complete content in delivery

Comment (Israel-Laguan) [reply src/pages/index.js]: yes

**Comment by elpollo12** [comment]
I have updated the code. Now the map should fit the available space and avoid scrolling @Israel-Laguan @AlejandroSalcido

**Review by Israel-Laguan — APPROVED** [review]
Lets face other fixes in other tickets

**Pull request merged**